patches/0002-remove_min_reviwers_defaults.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go b/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go
+index f51caa42..af9b2ae9 100644
+--- a/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go
++++ b/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go
+@@ -38,7 +38,6 @@ func ResourceBranchPolicyMinReviewers() *schema.Resource {
+ 	settingsSchema["on_last_iteration_require_vote"] = &schema.Schema{
+ 		Type:          schema.TypeBool,
+ 		Optional:      true,
+-		Default:       false,
+ 		ConflictsWith: []string{"settings.0.on_push_reset_approved_votes", "settings.0.on_push_reset_all_votes"},
```

Comment:
Ah so the ConflictsWith fixes it? I'm curious about the ".0" in the path. What's that? Does it refer to an element of a MaxItemsOne collection? Just making sure it's never "1,2,3" which would introduce more issues.

Reply:
Yeah, exactly - settings is a MaxItemsOne block, so .0 just gets the actual item there.